src/handlers/cost-code.js

```diff
@@ -1,22 +1,37 @@
 import Boomify from 'boom';
+import CostCode from '../models/costcode.js';
+
+const Joi = require('joi');
+
+const schema = Joi.object().keys({
```

Comment:
Move the validation out of this file and put it on the route itself. Something like https://hapijs.com/tutorials/validation

Reply:
https://futurestud.io/tutorials/hapi-request-payload-validation-with-joi

Reply:
Actually the more I think about it, put the schemas in the model that corrispondes to it and export that. You can delete the models I have.